solidity/Swarm.sol

```diff
@@ -1,31 +1,30 @@
 /*
```

Comment:
Same as before, anything that's been mentioned earlier and is still relevant here still counts!